tests/appsec/test_conf.py

```diff
@@ -101,14 +104,30 @@ def test_appsec_rules(self):
         """ test DD_APPSEC_RULES = custom rules file """
         interfaces.library.assert_waf_attack(self.r_appsec_rules, pattern="dedicated-value-for-testing-purpose")
 
+    def _find_series(self, request_type, namespace, metric):
```

Comment:
Is this used?

Reply:
was a left over from trying to use telemetry data.
removed